src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
```

Comment:
since you have already set `font-family` for `html`, you can use `font-family: inherit` for specific elements